services/provider/client/client.go

```diff
@@ -133,3 +133,54 @@ func (cc *OCSProviderClient) AcknowledgeOnboarding(ctx context.Context, consumer
 
 	return cc.Client.AcknowledgeOnboarding(apiCtx, req)
 }
+
+func (cc *OCSProviderClient) ProvisionStorageClassClaim(ctx context.Context, consumerUUID, claimUUID string, storageType pb.ProvisionStorageClassClaimRequest_StorageType) (*pb.ProvisionStorageClassClaimResponse, error) {
```

Comment:
The fact we are creating a storage class claim on the provider is an implementation detail from the perspective of the consumer cluster, so provisioning might be the wrong verb to use here. I will suggest:
`fulfilStorageClassClaim` instead. This way we are not hinting at a specific provider implementation

Reply:
for me it's not a problem, this was the names taken from the design discussions

Reply:
This is a public API so names are important.